.eslintrc

```diff
@@ -135,9 +134,7 @@
     "space-before-blocks": [2, "always"],
     "space-in-parens": [2, "never"],
     "space-return-throw-case": 2,
-    "spaced-comment": [2, "never"],
     "use-isnan": 2,
-    "valid-jsdoc": 2,
```

Comment:
lol dude their were no comments to begin with so having some is better than none